docs/product/size-analysis/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Size Analysis (Mobile)
+sidebar_order: 136
+description: Track and optimize your mobile app's binary size with automated uploads, detailed breakdowns, and build comparisons.
+---
+
+<Include name="size-analysis/ea" />
+
+Size Analysis helps you monitor and optimize your mobile application's binary size. By tracking your app size across builds and releases, you can catch unexpected size increases, understand what's contributing to your app's footprint, and deliver faster downloads for your users.
+
+## Why Track App Size?
+
+Mobile app size directly impacts user acquisition and retention. Smaller apps:
+
+- Download faster, reducing user drop-off during installation
+- Take up less storage space on user devices
+- Are more likely to be downloaded on slower networks
+- Update more quickly, improving user experience
+
+Size Analysis gives you visibility into your app's size trends and helps you catch regressions before they reach production.
+
+## How Size Analysis Works
+
+### Track Size Across Builds
+
+Upload your iOS (`.xcarchive`, `.ipa`) or Android (`.aab`, `.apk`) builds to Sentry as part of your CI pipeline. Size Analysis automatically extracts size metrics and tracks changes over time. [Get started with automated uploads](/product/size-analysis/uploading-builds/).
+
+### Compare Builds
+
+When you open a pull request, Size Analysis automatically compares your PR build against the base branch, showing you exactly how your changes impact app size. View the results directly in [sentry.io](https://sentry.io) or receive status checks in GitHub.
+
+### Understand Size Contributors
+
+Use the treemap and breakdown views in Sentry to see which components, frameworks, and resources contribute most to your app size so you can identify optimization opportunities quickly.
+
+### Integrate with Your Workflow
+
+Set up the [GitHub integration](/product/size-analysis/github-integration/) to receive size change notifications as PR comments and status checks, making size considerations a natural part of your code review process.
+
+## Supported Platforms
```

Comment:
Shall we mention something about the hybrids (RN, Flutter) that they also work through the native uploaded bundles?